backend/src/config/index.ts

```diff
@@ -67,7 +65,9 @@ const email = {
     process.env.EMAIL_LINK_VERIFICATION || 'http://localhost/checkEmail/{code}',
   EMAIL_LINK_SETPASSWORD:
     process.env.EMAIL_LINK_SETPASSWORD || 'http://localhost/reset-password/{code}',
-  RESEND_TIME: isNaN(resendTime) ? 10 : resendTime,
+  EMAIL_CODE_VALID_TIME: process.env.EMAIL_CODE_VALID_TIME
+    ? parseInt(process.env.EMAIL_CODE_VALID_TIME)
+    : 10,
```

Comment:
does this work for `EMAIL_CODE_VALID_TIME=abc` ?

Might be needed to do this:
```suggestion
  EMAIL_CODE_VALID_TIME: process.env.EMAIL_CODE_VALID_TIME
    ? parseInt(process.env.EMAIL_CODE_VALID_TIME) || 10
    : 10,
```